File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0,j=1,k=0;
        int n = nums.length, count =0;

        while(i<n && j<n){
            if(nums[i] == nums[j]){
                j++;
            }
            else{
                i++;
                nums[i] = nums[j];
                count++;
            }
        }
        return count+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        curr = 0
        rep = 1
        while rep < len(nums):
            if nums[curr] == nums[rep]:
                rep += 1
            else:
                curr += 1
                nums[curr] = nums[rep]
                rep += 1
        
        return curr+1

            
                